apollo-kotlin-sample/src/main/graphql/com/apollographql/apollo/kotlinsample/GithubRepositoriesQuery.graphql

```diff
@@ -0,0 +1,22 @@
+query GithubRepositories(
+  $repositoriesCount: Int!
+  $orderBy: RepositoryOrderField!
+  $orderDirection: OrderDirection!
+) {
+  viewer {
+    repositories(
```

Comment:
For some reason, this returns some repos where I'm not the owner but have admin rights. For these repos, I have an error when trying to display details.

Reply:
I've fixed it by limiting to repos where you're the owner.